### ISSUE TITLE
audio2audio tends to create louder result, often distorts.

### ISSUE THREAD

**Issue by pokepress** [ace-step/ACE-Step#287]
I've noticed that the audio2audio functionality tends to have the output louder than the input. I suspect this is a function of how the noise is generated and combined with the source audio:

`noise = torch.empty_like(zt_edit).normal_(generator=random_generators[0] if random_generators else None)`
`noisy_image = gt_latents * (1 - scheduler.sigma_max) + noise * scheduler.sigma_max`

Mathematically, the above is correct and should result in an altered waveform where all values are between -1 and 1. What I think has happened here is that although the average amplitude if the noise is 0.5, the average amplitude of the training data (especially in certain genres) is _above_ 0.5, so the model tends to make sounds louder. As a result, you get more distortion as the "Refer audio strength" increases. It might be possible to fix this by scaling the source audio so it has an average amplitude of 0.5 like the noise.
